spk/borgbackup/src/requirements-crossenv.txt

```diff
@@ -14,4 +14,6 @@ ruamel.yaml.clib==0.2.12
 #  PYO3_CROSS_INCLUDE_DIR=$(STAGING_INSTALL_PREFIX)/include/
 # Requires path to maturin from crossenv
 #  ENV += PATH=$(WORK_DIR)/crossenv/build/bin:$(PATH)
-rpds-py==0.20.1
+# rpds-py==0.21.0 is the latest version that builds with rust 1.77.2 (ARMv5, ARMv7L)
+# supporting carco lock version 3 (but not version 4)
+rpds-py==0.21.0
```

Comment:
This is another issue with rust.
Future updates of rpds-py will drop support for ARMv5 and will require actual rust toolchain for ARMv7L.